### PR TITLE
Document screenshots as common content

### PR DESCRIPTION
**Pull request by marandaneto**
https://github.com/getsentry/sentry-java/releases/tag/6.0.0-alpha.5

### CODE REVIEW

File: src/platforms/common/enriching-events/screenshots.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  - android
+---
+
+<Note>
+
+The Screenshots feature is available in Beta. Beta features are still in-progress and may have bugs. We recognize the irony.
+
+</Note>
+
+When a user experiences an error, Sentry provides the ability to take a screenshot and include as an [attachment](/platforms/android/enriching-events/attachments/).
```
Comment (marandaneto) [inline src/platforms/common/enriching-events/screenshots.mdx:15]: @imatwawana is there a way to link to `enriching-events/attachments` page but not from the `android` platform? rather redirecting to the current platform? this content will be available for other platforms as well.

Comment (imatwawana) [reply src/platforms/common/enriching-events/screenshots.mdx:15]: A PlatformLink component should work: 

<PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink>

Comment (imatwawana) [inline src/platforms/common/enriching-events/screenshots.mdx:15]:
```suggestion
When a user experiences an error, Sentry provides the ability to take a screenshot and include it as an <PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink>.
```

File: src/platforms/common/enriching-events/screenshots.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+This feature only applies to SDKs with a user interface, such as Mobile and Desktop. In some environments such as native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. So inherently this feature will be a best effort solution.
```
Comment (imatwawana) [inline src/platforms/common/enriching-events/screenshots.mdx:17]:
```suggestion
This feature only applies to SDKs with a user interface, such as the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. So inherently, this feature will be a best effort solution.
```

File: src/platforms/common/enriching-events/screenshots.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Enabling Screenshots
+
+Screenshots may contain [PII](/platforms/android/data-management/sensitive-data/) and is an opt-in feature, you can enable that by doing.
```
Comment (imatwawana) [inline src/platforms/common/enriching-events/screenshots.mdx:21]:
```suggestion
Screenshots may contain [PII](/platforms/android/data-management/sensitive-data/) and is an opt-in feature. You can enable it as shown below:
```

File: src/platforms/common/enriching-events/screenshots.mdx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+<PlatformContent includePath="enriching-events/attach-screenshots" />
+
+## Viewing Screenshots
+
+Screenshots appear on the **Issue Details** page, where you can view the thumbnail, as well as associated events. Click the Event ID to open the **Issue Details** of that specific event.
```
Comment (imatwawana) [inline src/platforms/common/enriching-events/screenshots.mdx:27]:
```suggestion
Screenshots appear on the **Issue Details** page, where you can view the thumbnail, as well as associated events. Click the event ID to open the **Issue Details** of that specific event.
```

File: src/platforms/common/enriching-events/screenshots.mdx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+![Screenshots Thumbnail Example](screenshots/screenshot-thumbnail.png)
+
+Alternately, screenshots also appear in the _Attachments_ tab on the **Issue Details** page, where you can view all attachments, as well as associated events. Click the Event ID to open the **Issue Details** of that specific event.
```
Comment (imatwawana) [inline src/platforms/common/enriching-events/screenshots.mdx:31]:
```suggestion
Alternatively, screenshots are also available in the "Attachments" tab on the **Issue Details** page, where you can view all attachments, as well as associated events. Click the event ID to open the **Issue Details** of that specific event.
```

**Review by imatwawana — APPROVED** [review]

File: src/platforms/common/enriching-events/screenshots.mdx [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+</Note>
+
+When a user experiences an error, Sentry provides the ability to take a screenshot and include it as an <PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink>.
+
+This feature only applies to SDKs with a user interface, such as the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. So inherently, this feature will be a best effort solution.
+
+## Enabling Screenshots
+
+Screenshots may contain [PII](/platforms/android/data-management/sensitive-data/) and is an opt-in feature. You can enable it as shown below:
```
Comment (marandaneto) [inline src/platforms/common/enriching-events/screenshots.mdx:21]:
```suggestion
Screenshots may contain <PlatformLink to="/data-management/sensitive-data/">PII</PlatformLink>[PII] and is an opt-in feature. You can enable it as shown below:
```

**Review by bitsandfoxes — APPROVED** [review]

**Pull request merged**